frontend/src/axios.js

```diff
@@ -0,0 +1,4 @@
+import axios from 'axios';
+
+axios.defaults.baseURL = 'http://localhost:8000/';
```

Comment:
It'd probably be better to have this as a configurable environment variable instead that is set during build time. This means that we can set the appropriate URL automatically into the production container build pipelines.